apps/server/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "server",
```

Comment:
use "@project-name/package-name" to name a package. ex: @crud-beer/server

Reply:
Fixed